docs/alerts/monitors/settings.md

```diff
@@ -115,6 +117,21 @@ To add a tag while creating a new monitor:
    * Apply an existing tag by selecting the key and its possible values from the dropdown.
 1. Click **Save**.
 
+
+### What happens next?
+
+Tags will appear on all future alerts triggered by the same monitor. Tags will *not* be applied to any past alerts triggered by this monitor.
+
+To view and edit tags, go to your alerts list page. <!--Kanso [**Classic UI**](/docs/get-started/sumo-logic-ui/). Kanso--> In the main Sumo Logic menu, select **Manage Data > Monitoring > Monitors**. <!--Kanso <br/>[**New UI**](/docs/get-started/sumo-logic-ui-new/). In the main Sumo Logic menu, select **Alerts > Monitors**. You can also click the **Go To...** menu at the top of the screen and select **Monitors**. Kanso-->
```

Comment:
navigating via `Manage Data > Monitoring > Monitors` will take the user to the Monitors Page and not the Alert List page as we have mentioned in the previous line.